nltk/translate/util.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+
+def jacknifing(score_list, averaging=True):
+        ''' This is a averaging function for calculating ROUGE score
+    when multiple references are present. This technique has been
+    referred to as the Jacknifing technique in the original paper.
+    score_list = list of scores over which averaging occurs
+    '''
+        if(len(score_list) == 1):
+            return np.mean(score_list)
+        elif((len(score_list) > 1) and (averaging is False)):
+            return score_list
+        else:
+            average = []
+            for i in score_list:
+                dummy = [j for j in score_list if i != j]
+                average.append(max(dummy))
+            return(np.mean(average))
+
+
+def lcs(X, Y, m, n):
+        '''This function returns the length of the
+        longest common subsequence(LCS) between two
+        given strings and also the LCS.
+        X = list containing all the tokens of the first string
+        Y = list containing all the tokens of the second string
+        m = no. of tokens of the first string
+        n = no.of tokens of the second string
+
+        Note: Tokenization of the strings can be done at both
+        character-level and word-level.
+        '''
```

Comment:
The naming conventions in this function isn't good. We have `L` undefined and then we have `lcs` being a array instead the `lcs()` function.

Reply:
Why didn't you follow the pseudo-code convention in the http://www.aclweb.org/anthology/W04-1013 paper? You can easily use remove the weighted option from https://github.com/nltk/nltk/issues/1993#issuecomment-376755814 too =)

Reply:
Followed and completed this part